test/lib/rubycritic/analysers/smells/reek_test.rb

```diff
@@ -17,6 +17,11 @@
       smell = @analysed_module.smells.first
       smell.message.must_equal "has boolean parameter 'reek'"
     end
+
+    it "has documentation link" do
+      smell = @analysed_module.smells.first
+      smell.documentation.must_equal "https://github.com/troessner/reek/wiki/Boolean-Parameter"
```

Comment:
There’s an age-old discussion whether tests should reuse parts of the code under test, and one side says that not reusing the `REEK_WIKI` constant is actually better, becasue it tests it as well (treats the constant as an implementation detail of `ReekSmells`). :)